integration_tests/src/assembly/bin.xml

```diff
@@ -50,6 +50,10 @@
             <source>${project.build.outputDirectory}/rapids4spark-version-info.properties</source>
             <outputDirectory>integration_tests</outputDirectory>
         </file>
+        <file>
+            <source>${spark.rapids.source.basedir}/integration_tests/target/spark-rapids-default-configs.json</source>
```

Comment:
```suggestion
            <source>${project.build.outputDirectory}/spark-rapids-default-configs.json</source>
```

Reply:
Hmm I thought `project.build.outputDirectory` is `integration_tests/target/classes` not `integration_tests/target`?

Reply:
you can use `${project.build.directory}`

Reply:
Yes, sorry, picked the wrong build directory variable.  I just know that hardcoding `/target` is not what we should be doing, as there's a variable for that _somewhere_. :smile:

Reply:
Updated as suggested.